### DISCUSSION TITLE
QDate text-color for speical dates (holiday)

### DISCUSSION THREAD

**Discussion by webasoo** [quasarframework/quasar#15742] [Ideas / Proposals]
A Feature in QDate component that can set different color for specific dates.
or that can set holiday days and the component shown them different.
thanks

**Comment by metalsadman** [comment]
there's 
![image](https://user-images.githubusercontent.com/31038838/234180236-df1b6a55-65f1-4185-a5c8-b64be67b8d0a.png)

**Reply by webasoo** [reply]
thanks. But this changes the color of the date's underline. while I need change **text-color**.
(also, events may different with holidays. thus in some of use-cases, events exist in holidays dates and we may need both together )
